openid-4-verifiable-presentations-1_0.md

```diff
@@ -674,6 +710,14 @@ Note that Relying Parties must verify that the issuer of a received presentation
 trusted on their own and this feature mainly aims to help data minimization by not
 revealing information that would likely be rejected.
 
+`allow_replay`:
```

Comment:
sounds weird, no one is interested in allowing replay - I would rather call this element "key_binding_required"

Reply:
Making this sound slightly weird and not like something you want to use was exactly the intention. I fear that Verifiers may just use it because "I don't need this weird proof thing anyway".

Reply:
renamed to `require_cryptographic_holder_binding`